src/app/pages/challenges/challenge-view/challenge-view.component.ts

```diff
@@ -14,22 +20,77 @@ import { PageTitleService } from '@shared/page-title/page-title.service';
   styleUrls: ['./challenge-view.component.scss'],
 })
 export class ChallengeViewComponent implements OnInit {
-  challenge$!: Observable<Challenge>;
+
+  @Input() selected!: boolean;
+  @Output() selectedChange = new EventEmitter<boolean>();
+
+  // challenge$!: Observable<Challenge>;
+  challenge!: Challenge;
+  personList: Person[] = [];
+  orgList: Organization[] = [];
+  progressValue: number = 0;
 
   constructor(
     private router: Router,
     private route: ActivatedRoute,
     private challengeService: ChallengeService,
+    private organizationService: OrganizationService,
+    private personService: PersonService,
     private pageTitleService: PageTitleService
   ) {}
 
+
   ngOnInit(): void {
-    this.challenge$ = this.route.params.pipe(
-      switchMap((params) => this.challengeService.getChallenge(params.id))
+    // this.challenge$ = this.route.params.pipe(
+    //   switchMap((params) => this.challengeService.getChallenge(params.id))
+    // );
+    // this.challenge$.subscribe(challenge => {
+    //   this.pageTitleService.setTitle(challenge.name);
+    // });
+    this.route.params.pipe(
+      switchMap((params) => this.challengeService.getChallenge(params.id)),
+      tap(challengeRes => {
+        this.challenge = challengeRes;
+        this.pageTitleService.setTitle(challengeRes.name);
+      }),
+      switchMap(challenge => 
+        forkJoin(
+          challenge.organizerIds
+            .map(organizerId => this.personService.getPerson(organizerId))
+        )
+      ),
+      tap((persons)=> this.personList = persons),
+      switchMap((persons) => {
+        const orgIds = [...new Set(persons.flatMap(person => person.organizationIds))]
```

Comment:
Why not use `Challenge.organizations`? The issue with the current implementation is that 10 years from now, the organizers will be working for other organization and the name of these orgs will be displayed on the page of old challenges, while these organizations have in no way contributed to the organization of the challenge.

Reply:
I just noticed that `Challenge.organizations` is not (yet) a thing. This property will be added in an update of the schemas coming this week. For now let's not show organizations on the challenge page for simplicity.